pennylane/transforms/hamiltonian_expand.py

```diff
@@ -77,6 +100,25 @@ def hamiltonian_expand(tape):
         raise ValueError(
             "Passed tape must end in `qml.expval(H)`, where H is of type `qml.Hamiltonian`"
         )
-
-    hamiltonian.simplify()
-    return qml.transforms.measurement_grouping(tape, hamiltonian.ops, hamiltonian.coeffs)
+    if group:
+        hamiltonian.simplify()
+        return qml.transforms.measurement_grouping(tape, hamiltonian.ops, hamiltonian.coeffs)
+
+    # create tapes that measure the Pauli-words in the Hamiltonian
+    tapes = []
+    for ob in hamiltonian.ops:
```

Comment:
`Hamiltonian.ops` - our terminology is all over the place 😆

Reply:
Yep :)